crates/toolchain/instructions/src/riscv.rs

```diff
@@ -5,3 +5,4 @@ pub const RV32_CELL_BITS: usize = 8;
 pub const RV32_IMM_AS: u32 = 0;
 pub const RV32_REGISTER_AS: u32 = 1;
 pub const RV32_MEMORY_AS: u32 = 2;
+pub const NATIVE_KERNEL_AS: u32 = 5;
```

Comment:
it's (1) weird this is in riscv.rs, (2) we prefer not to have these constants in `openvm-instructions` since its more specific to a certain extension. Arguably this whole file should be removed from `openvm-instructions`, but I think it exists for some archaic reasons